### PR TITLE
Panel: fixed $shortFilename callback typehint

### PR DESCRIPTION
**Pull request by matej21**

### CODE REVIEW

File: src/Kdyby/Events/Diagnostics/Panel.php [hunk 1]
```diff
@@ -388,7 +388,7 @@ private function renderListeners($ids)
 
 		$h = 'htmlspecialchars';
 
-		$shortFilename = function (\ReflectionMethod $refl) {
+		$shortFilename = function (\ReflectionFunctionAbstract $refl) {
```
Comment (fprochazka) [inline src/Kdyby/Events/Diagnostics/Panel.php]: Maybe simply `\Reflector` would be enough?

Comment (matej21) [inline src/Kdyby/Events/Diagnostics/Panel.php]: Callback::toReflection returns only ReflectionMethod or ReflectionFunction. And Reflector do not have getFilename, getStartLine etc.

Comment (fprochazka) [inline src/Kdyby/Events/Diagnostics/Panel.php]: You're right :)

**Comment by fprochazka** [comment]
Thank you!

**Pull request merged**